app/src/main/java/com/dicoding/picodiploma/sejiwaproject/features/league/detail/page/SectionsPagerAdapter.kt

```diff
@@ -0,0 +1,25 @@
+package com.dicoding.picodiploma.sejiwaproject.features.league.detail.page
+
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.FragmentManager
+import androidx.fragment.app.FragmentPagerAdapter
+
+/**
+ * A [FragmentPagerAdapter] that returns a fragment corresponding to
+ * one of the sections/tabs/pages.
+ */
+class SectionsPagerAdapter(fm: FragmentManager) : FragmentPagerAdapter(fm) {
+
+    val fragmentList = arrayListOf<Fragment>()
+    val fragmentTitle = arrayListOf<String>()
```

Comment:
Functions or variables that are only used in the same class should be made private.

Reply:
resolved, thanks